consts.php

```diff
@@ -59,6 +63,17 @@
     'recipe-blog',
 ];
 
+// lockstepped repos that are "three less" e.g. silverstripe/admin 1 is for CMS 4
+const LOCKSTEPPED_REPOS_VERSION_IS_THREE_LESS = [
+    'silverstripe-admin',
+    'silverstripe-asset-admin',
+    'silverstripe-assets',
+    'silverstripe-campaign-admin',
+    'silverstripe-errorpage',
+    'silverstripe-versioned',
+    'silverstripe-versioned-admin',
+];
```

Comment:
This seems unrelated - there's no information anywhere in this PR or in the related issue that tells me what you're doing here or why. Please provide an appropriate level of information in the PR description so that I can review this PR.

Reply:
It's used on line 54 of job_creator.php

Reply:
We talked offline about this. One of the unit tests that is being updated in this PR was incorrect, and this along with the changes in job_creator.php resolve the problem with that test.